### PR TITLE
Central Apollyon Rework

### PR DESCRIPTION
**Pull request by jmcmorris**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

Migrate and rework Central Apollyon Limbus area to be in the new battlefield framework. All of this work is heavily based off of the following captures and sources:
https://www.youtube.com/watch?v=lJV2juzuDQQ
https://www.youtube.com/watch?v=qk-f6a1kBBg
https://ffxiclopedia.fandom.com/wiki/Central_Apollyon
https://www.bg-wiki.com/ffxi/Central_Apollyon

A huge chunk of these changes is dealing with mobskills. Previously Omega skills had lots of conditional checks happening to determine when it could use a skill. That logic has been stripped out and it's now simply changing the skill list when changing forms.

## Steps to test these changes

```
!addkeyitem black_card
!addkeyitem cosmo_cleanse
!additem 1909
!additem 1910
!additem 1987
!additem 1988
!pos 600 -0.5 -600 38
```
Play through the area following the above guides and captures.

### CODE REVIEW

**Comment by jmcmorris** [comment]
This is mostly just waiting for #2940 to make it through since they both touch mobskills.

**Review by claywar — APPROVED** [review]

File: scripts/globals/battlefield.lua [hunk 1]
```diff
@@ -82,6 +82,7 @@ xi.battlefield.id =
     NE_APOLLYON = 1292,
     SE_APOLLYON = 1293,
     CS_APOLLYON = 1294,
+    CENTRAL_APOLLYON = 1296,
```
Comment (WinterSolstice8) [inline scripts/globals/battlefield.lua]: CI didn't trip for alignment here? interesting.

Comment (claywar) [reply scripts/globals/battlefield.lua]: Would be something nice to have added in the future, but I think that check is only on the cpp side for now.

Comment (jmcmorris) [reply scripts/globals/battlefield.lua]: I can fix that alignment in just a few

Comment (jmcmorris) [reply scripts/globals/battlefield.lua]: Actually, would you guys be opposed if I just filled out the entire thing with all existing battlefields? I can easily do this and would be much easier than adding them one at a time.

Comment (WinterSolstice8) [reply scripts/globals/battlefield.lua]: works for me

**Review by WinterSolstice8 — APPROVED** [review]

**Pull request merged**